giskard/models/model_explanation.py

```diff
@@ -303,9 +306,12 @@ def text_explanation_prediction_wrapper(
 ) -> Callable:
     def text_predict(text_documents: List[str]):
         num_documents = len(text_documents)
-
-        df_with_text_documents = input_example.append([input_example] * (num_documents - 1), ignore_index=True)
-        df_with_text_documents[text_column] = pd.DataFrame(text_documents)
+        df_with_text_documents = (
+            input_example.copy(deep=True)
```

Comment:
not sure we need the `deep=True`

Reply:
Yeah, true. I put it because I had some issued to debug